src/scrape_up/flyrobu/flyrobu.py

```diff
@@ -157,3 +158,84 @@ def search(self, keyword):
 
         result = self.outputs
         return result
+
+    def get_product_details(self, product_name):
+      try:
```

Comment:
Add docstring with proper code sample and example.

Reply:
DOne